src/workerd/server/server.c++

```diff
@@ -1415,8 +1415,8 @@ private:
 
   kj::Own<ActorChannel> getGlobalActor(uint channel, const ActorIdFactory::ActorId& id,
       kj::Maybe<kj::String> locationHint, ActorGetMode mode) override {
-    JSG_REQUIRE(mode == ActorGetMode::GET_OR_CREATE, "workerd only supports GET_OR_CREATE mode "
-        "for getting actor stubs");
+    JSG_REQUIRE(mode == ActorGetMode::GET_OR_CREATE, TypeError,
```

Comment:
Non-blocking Super minor Nit: I'd make this just Error instead of TypeError

Reply:
Ah, awesome, thanks!  Will update.